command.php

```diff
@@ -4,6 +4,8 @@
 	return;
 }
 
+require 'vendor/autoload.php';
```

Comment:
You need to check for the existence of this file first, because the folder will not exist if the package is used as a dependency, not a root package:

```php
$autoload = dirname( __FILE__ ) . '/vendor/autoload.php';
if ( file_exists( $autoload ) ) {
	require_once $autoload;
}
```

Reply:
My apologies. I made the necessary changes.